cypress/support/fragments/orders/orderLines.js

```diff
@@ -190,10 +191,10 @@ export default {
   POLineInfodorPhysicalMaterial: (orderLineTitleName) => {
     cy.do([
       orderLineTitleField.fillIn(orderLineTitleName),
-      orderFormatSelect.choose('Physical resource'),
+      orderFormatSelect.choose(ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE),
       acquisitionMethodButton.click(),
-      SelectionOption('Depository').click(),
-      receivingWorkflowSelect.choose('Synchronized order and receipt quantity'),
+      SelectionOption(ACQUISITION_METHOD_NAMES.DEPOSITORY).click(),
+      receivingWorkflowSelect.choose(RECEIVING_WORKFLOW_NAMES.SYNCHRONIZED_ORDER_AND_RECEIPT_QUANTITY),
       physicalUnitPriceTextField.fillIn(physicalUnitPrice),
       quantityPhysicalTextField.fillIn(quantityPhysical),
       materialTypeSelect.choose('book'),
```

Comment:
Please use here the variable from the constants file from the support directory

Reply:
Done